_includes/master/manifests/calico-etcd-secrets.yaml

```diff
@@ -7,10 +7,10 @@ metadata:
   name: calico-etcd-secrets
   namespace: kube-system
 data:
-  # Populate the following files with etcd TLS configuration if desired, but leave blank if
+  # Populate the following with etcd TLS configuration if desired, but leave blank if
   # not using TLS for etcd.
-  # This self-hosted install expects three files with the following names.  The values
-  # should be base64 encoded strings of the entire contents of each file.
+  # The values below should be populated with base64 encoded strings of the entire
+  # contents of each file that associated with the TLS data.
```

Comment:
> that associated with the TLS data.

I think this is worded a bit odd, otherwise LGTM.

Reply:
Yeah agreed.
What do you think about including an example base64 command, like in the docs `cat <file> | base64 -w 0`?

Reply:
so long as it's a simple one-liner it sounds OK to me.